### PR TITLE
Changes from live version(Cluster, coverage, density, etc)

### PR DESCRIPTION
**Pull request by MichaelGerg**

### CODE REVIEW

File: src/Cluster/coverage.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#$ -V
+#$ -N sequdas_coverage
+#$ -cwd
+#$ -pe smp 25
```
Comment (dfornika) [inline src/Cluster/coverage.sh]: Is it necessary to request 25 slots for this job?

File: src/Cluster/fastqc.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#$ -V
+#$ -N sequdas_fastqc
+#$ -cwd
+#$ -pe smp 25
```
Comment (dfornika) [inline src/Cluster/fastqc.sh]: Is it necessary to request 25 slots for this job?

File: src/Cluster/kraken2.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#$ -V
+#$ -N sequdas_kraken2
+#$ -cwd
+#$ -pe smp 25
```
Comment (dfornika) [inline src/Cluster/kraken2.sh]: Is it necessary to request 25 slots for this job?

File: src/Cluster/krona.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#$ -V
+#$ -N sequdas_krona
+#$ -cwd
+#$ -pe smp 25
```
Comment (dfornika) [inline src/Cluster/krona.sh]: Is it necessary to request 25 slots for this job?

File: src/Cluster/coverage.sh (hunk 1 above)
Comment (MichaelGerg) [reply src/Cluster/coverage.sh]: Right, I forgot to change all of the files.

I changed them all to 7

Thanks Dan!

**Comment by dfornika** [comment]
Looking good. Are you planning on adding any more changes to this pull-request?

**Pull request merged**